XRTK-Core/Packages/com.xrtk.core/Extensions/TransformExtensions.cs

```diff
@@ -115,7 +116,10 @@ public static Bounds GetColliderBounds(this Transform transform, bool syncTransf
                 Physics.SyncTransforms(); // Update collider bounds
             }
 
-            var colliders = transform.GetComponentsInChildren<Collider>();
+            if (colliders == null)
```

Comment:
mega nit: null coalescing operators?

Reply:
[Shouldn't use them on `UnityEngine.Object` types](https://www.google.com/url?sa=t&source=web&rct=j&url=https://github.com/JetBrains/resharper-unity/wiki/Possible-unintended-bypass-of-lifetime-check-of-underlying-Unity-engine-object&ved=2ahUKEwjy89adkajnAhXKK80KHdElBhEQFjAAegQIBxAC&usg=AOvVaw0ZKb6-1ywLRICS3m1GgqrZ)